packages/EasyCore/src/Bridge/Symfony/Traits/InvalidDataGeneratorTrait.php

```diff
@@ -0,0 +1,535 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyCore\Bridge\Symfony\Traits;
+
+use Carbon\Carbon;
+use Doctrine\Common\Collections\ArrayCollection;
+
+trait InvalidDataGeneratorTrait
+{
+    /**
+     * @param mixed[] $cases
+     *
+     * @return mixed[]
+     */
+    public function asString(array $cases): array
+    {
+        return \array_map(static function (array $case): array {
+            $case['data'] = \array_map(static function ($value) {
+                return (string)$value;
+            }, $case['data']);
+
+            return $case;
+        }, $cases);
+    }
+
+    /**
+     * @return mixed[]
+     */
+    public function getArrayCollectionWithFewerItems(string $property, int $minElements): array
+    {
+        $fewerElements = \array_fill(0, $minElements - 1, null);
+        $fewerArrayCollection = new ArrayCollection($fewerElements);
+
+        return [
+            "${property} has too few elements in the collection" => [
+                'data' => [
+                    $property => $fewerArrayCollection,
+                ],
+                'message' => "This collection should contain ${minElements} element or more.",
+            ],
+        ];
+    }
+
+    /**
+     * @return mixed[]
+     */
+    public function getArrayWithFewerItems(string $property, int $minElements): array
+    {
+        return [
+            "${property} has too few elements in the array" => [
+                'data' => [
+                    $property => \array_fill(0, $minElements - 1, null),
+                ],
+                'message' => "This collection should contain ${minElements} element or more.",
+            ],
+        ];
+    }
+
+    /**
+     * @return mixed[]
+     */
+    public function getArrayWithMoreItems(string $property, int $maxElements): array
+    {
+        return [
+            "${property} has too many elements in the collection" => [
+                'data' => [
+                    $property => \array_fill(0, $maxElements + 1, []),
+                ],
+                'message' => "This collection should contain ${maxElements} elements or less.",
+            ],
+        ];
+    }
+
+    /**
+     * @return mixed[]
+     */
+    public function getBlankString(string $property): array
+    {
+        return [
+            "${property} is blank" => [
+                'data' => [
+                    $property => '',
+                ],
+                'message' => 'This value should not be blank.',
+            ],
+        ];
+    }
+
+    /**
+     * @return mixed[]
+     */
+    public function getBlankStringInArray(string $property): array
```

Comment:
Can't we use `getBlankString` + `wrapWith` instead of introducing a separate method for this?